### PR TITLE
Update REST API paths for Service Hooks Subscriptions, and Project ID…

### LINKED ISSUE #1468: Hyperlinks to "Subscriptions" and "Project REST API" result in page not found error.
Two hyperlinks located in the [Create a service hooks subscription programmatically](https://docs.microsoft.com/en-us/vsts/service-hooks/create-subscription?view=vsts) page result in a 404 page not found error. 

1. [Subscriptions](https://visualstudio.microsoft.com/docs/integrate/api/hooks/subscriptions) hyperlink: Located at the beginning of the documentation.

2. [Project REST API](https://visualstudio.microsoft.com/en-us/docs/integrate/api/tfs/overview) hyperlink: Located under the first bullet point within the "Before you begin" section.

### PR DESCRIPTION
**Pull request by IKwiotek**
… hyperlinks. The existing hyperlinks result in a page not found error. Fixes #1468

### CODE REVIEW

File: docs/service-hooks/create-subscription.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ ms.date: 08/04/2016
 
 # Create a service hooks subscription programmatically
 
-Using the [Subscriptions](https://visualstudio.microsoft.com/en-us/docs/integrate/api/hooks/subscriptions) REST APIs, you can programmatically create a subscription that performs an action on an external (consumer) service when a specific event occurs in a project. For example, you can create a subscription to notify your service when a build fails.
+Using the [Subscriptions](https://docs.microsoft.com/en-us/rest/api/vsts/hooks/subscriptions) REST APIs, you can programmatically create a subscription that performs an action on an external (consumer) service when a specific event occurs in a project. For example, you can create a subscription to notify your service when a build fails.
```
Comment (nschonni) [inline docs/service-hooks/create-subscription.md]: You can even drop the `https://docs.microsoft.com/en-us` since they're site root links

Comment (IKwiotek) [reply docs/service-hooks/create-subscription.md]: Thank you for the suggestion! I have updated the hyperlinks with relative links that reference the corresponding markdown files.

**Comment by steved0x** [comment]
@IKwiotek Thank you for your contribution, I am merging it now.

**Comment by IKwiotek** [comment]
@steved0x 
Do you happen to know if these changes were promoted to production? The links are presently broken and this PR would allow the respective pages to resolve correctly.
Thanks!

**Pull request merged**